scripts/variantstore/docs/aou/AOU_DELIVERABLES.md

```diff
@@ -52,26 +50,32 @@
+   - inputs to the python script are:
+     1. `--avro-path`: the directory path at which exported GVS Avro files are found in GCP
+     2. `--vds-path`: the desired output path to which the VDS should be written
+     3. `--temp-path`: a convenient path to temporary directory (we suggest the GCP bucket of the workspace that the notebook is in) 
+9. `GvsPrepareRangesCallset` workflow (Hail VDS output -- for callset stats)
+   - This workflow transforms the data in the vet tables into a schema optimized for VCF generation during the Extract step.
+   - It will need to be run once with `only_output_vet_tables` set to "true" (see [naming conventions doc](https://docs.google.com/document/d/1pNtuv7uDoiOFPbwe4zx5sAGH7MyxwKqXkyrpNmBxeow) for guidance on what to use for `extract_table_prefix` or cohort prefix, which you will need to keep track of for the callset stats); the default value is `false`.
    - This workflow does not use the Terra Data Entity Model to run, so be sure to select the `Run workflow with inputs defined by file paths` workflow submission option.
-9. **TBD VDS Prepare WDL/notebook/??**
-10. **TBD VDS Extract WDL/notebook/??**
-11. Run the notebook to create callset stats (**TODO**, see [VS-388](https://broadworkbench.atlassian.net/browse/VS-388)), for which you need
-     - "BigQuery Data Viewer" role for your @pmi-ops proxy group on the `spec-ops-aou:gvs_public_reference_data.gnomad_v3_sites` table
-     - the Google project ID you used for all the GVS WDLs (`project_id` input)
-     - the name of the BigQuery dataset you used for all the GVS WDLs (`dataset_name` input)
-     - the `extract_table_prefix` input from `GvsExtractCallset` step
-     - the `filter_set_name` input from `GvsCreateFilterSet` step
-12. [GvsCalculatePrecisionAndSensitivity](../../tieout/AoU_PRECISION_SENSITIVITY.md#generating-callset-precision-and-sensitivity-values) workflow, for which you need
-    - "Storage Object View" access granted for your @pmi-ops proxy group on the `gs://broad-dsp-spec-ops/gvs/truth` directory
-13. `GvsCallsetCost`
-    - The cost from this callset, which represents the total BigQuery cost (which is not represented in the Terra UI total workflow cost) from the GVS pipeline workflows, is used to calculate the cost of the callset as a whole and by sample.
+10. Run the notebook to create callset stats (**TODO**, see [VS-388](https://broadworkbench.atlassian.net/browse/VS-388)), for which you need
+   - "BigQuery Data Viewer" role for your @pmi-ops proxy group on the `spec-ops-aou:gvs_public_reference_data.gnomad_v3_sites` table
+   - the Google project ID you used for all the GVS WDLs (`project_id` input)
+   - the name of the BigQuery dataset you used for all the GVS WDLs (`dataset_name` input)
+   - the `extract_table_prefix` input from `GvsExtractCallset` step
+   - the `filter_set_name` input from `GvsCreateFilterSet` step
```

Comment:
nit: the formatting/structure for these should be the same as the other workflows